### DISCUSSION TITLE
List ALL defaults & possibly documentation?

### DISCUSSION THREAD

**Discussion by andromidasj** [tteck/Proxmox#3279] [Ideas]
I'm loving these scripts, and have been getting VMs in my Proxmox server moved over to LXCs and VMs from these helper scripts. Absolutely amazing.

That being said, it'd be awesome to know what the defaults are when running a new script just by glancing at the description. Some are listed in the description, like `Default Settings: 2GB RAM - 4GB Storage - 2vCPU`, but what about all the other types of configurations? Especially Privileged/Unprivileged. We can (and should) obviously check the script ourselves before running (where we'd find the defaults object that gives us all this info), but I think it'd also be a good idea to have this information in the description at a glance.

Generally, I think the descriptions could probably be a bit more detailed. For example, in the Plex LXC script, it says `With Privileged/Unprivileged Hardware Acceleration Support`. So does that mean the LXC will be Privileged or Unprivileged? What are the prerequisites to getting this set up if I use one over the other? If there were a link to discussions or documentation about this specific script, that'd probably also be a great way to reduce the same questions being asked in the discussions.

Again, these scripts are amazing and I'm excited to explore more of them!

**Comment by tteck** [comment]
Default settings now shows if the container is privileged

![Screenshot from 2024-06-28 06-49-48](https://github.com/tteck/Proxmox/assets/21342670/37fdaaac-bef4-470a-b88a-b282e7f54907)
